src/constants.js

```diff
@@ -3,10 +3,13 @@ import path from "path";
 import os from "os";
 
 export const NPM_TOKEN = typeof process.env.NODE_SECURE_TOKEN === "string" ? { token: process.env.NODE_SECURE_TOKEN } : {};
+export const SNYK_ORG = process.env.SNYK_ORG || "8327bf87-23c7-46c0-84c6-e46d38685d68";
```

Comment:
We need to remove default ORG and default TOKEN here (we don't want to publish them on github).

Reply:
yep, forgot to remove mocks